conf/messages

```diff
@@ -592,7 +592,7 @@ hts.reminder-confirmation.h2=What happens next
 hts.reminder-frequency.select-an-option=Select an option for setting up reminders
 reminder-select.reminderFrequency.reminder_frequency_empty=Select an option for setting up reminders
 
-hts.reminder-dashboard.title.h1=Reminders dashboard
+hts.reminder-dashboard.title.h1=Change or cancel email reminders
```

Comment:
As per the fix mentioned in the ticket the message to be updated is "Cancel or change email reminders"